Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/DriverStats.hs

```diff
@@ -23,6 +23,8 @@ data DriverStats = DriverStats
   { driverId :: Id Driver,
     idleSince :: UTCTime,
     totalRides :: Int,
-    totalDistance :: Meters
+    totalDistance :: Meters,
+    ridesCancelled :: Maybe Int,
+    totalRidesAssigned :: Maybe Int
```

Comment:
Difference b/w totalRide and totalRidesAssigned?

Reply:
Total ride is actually total completed rides for some reason 🙃. So can't use that.